packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -228,22 +238,31 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({
   const showImageCaptureModal = useCallback(() => {
     const close = showModal('capture-photo-modal', {
       saveFile: (file: UploadedFile) => {
-        setValue('image', file);
+        if (file) {
+          const filteredFiles = allowedImageTypes.includes('.' + file.fileName.split('.').pop().toLowerCase());
+          if (filteredFiles) {
+            setUploadedImages([...uploadedImages, file]);
+          } else {
+            console.error('Error: Unsupported file type');
```

Comment:
Errors should be displayed as some kind of notification (either an inline or snackbar) and not simply logged to the console.

Reply:
Probably want to use [reportError](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md#reporterror) for things like this

Reply:
In this case, because we want to show an `InlineNotification` similar to https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-attachments-app/src/camera-media-uploader/media-uploader.component.tsx#L81 here because they're both doing the same thing. So, using an `InlineNotification` is the ideal approach. We shouldn't log anything to the console here.